README.md

```diff
@@ -197,6 +198,61 @@ $ oc start-build example-quarkus-binary --from-dir=target
 And that's it. In a couple minutes you should have your Kogito Service application up and running using the same binaries
 built locally.
 
+### Build From File
+
+Kogito Operator can configure a build for you that will pull and build your application through the source to image (s2i)
+feature (available only on OpenShift). This kind of build consists to upload a Kogito service file, e.g. a dmn or a bpmn file
+to the OpenShift Cluster and trigger a new Source to Image build automatically.
+
+You can provide a single dmn, drl, bpmn or bpmn2 files, the standalone file can be locally on the filesystem or on a Web Site, like Git Hub:
+
+
+```bash
+# from filesystem
+$ kogito deploy-service example-dmn-quarkus --from-file TrafficViolation.dmn
```

Comment:
I think it would be good to use the same source for filesystem and from git repository. For example, the TrafficViolation.dmn file does not exist, but "Traffic Violation.dmn".

Reply:
Maybe for windows users?
At least on Unix systems it is not common to use name files with spaces.

We can change it.

@radtriste @sutaakar what do you guys think?

Reply:
> I think it would be good to use the same source for filesystem and from git repository

I didn't understand this question. Doesn't matter if the file has space or not, the plugin will create the project anyway. What we can do is run a regexp in the file name and emit a warn or deny the upload based on this rule.

Reply:
Oh you guys are talking about the README description of the example. Right, agreed.

Reply:
using spaces from now.